soc/arm/nxp_imx/rt/Kconfig.defconfig.series

```diff
@@ -40,7 +40,11 @@ endif
 # set the tick per sec as a divider of the GPT clock source
 config SYS_CLOCK_TICKS_PER_SEC
 	default 4096 if MCUX_GPT_TIMER
+
+DT_SYSCLK_PATH := $(dt_nodelabel_path,sysclk)
+
 config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default $(dt_node_int_prop_int,$(DT_SYSCLK_PATH),clock-frequency) if SOC_SERIES_IMX_RT10XX && CORTEX_M_SYSTICK
 	default 32768 if MCUX_GPT_TIMER
```

Comment:
The 32KHz clock can be derived from the RTC_OSC frequency of 32KHz, as that is where the CCM sources the low frequency reference clock from

Reply:
I guess I would need to have that clock in DT first, for both rt10xx (#49776) and then for rt11xx as well.

Reply:
Sure-maybe something to address in the future, since these changes are only present on the RT10xx right now.